src/Routers/FilesRouter.js

```diff
@@ -36,17 +36,31 @@ export class FilesRouter {
     const config = new Config(req.params.appId);
     const filesController = config.filesController;
     const filename = req.params.filename;
-    filesController.getFileData(config, filename).then((data) => {
-      res.status(200);
-      var contentType = mime.lookup(filename);
-      res.set('Content-Type', contentType);
-      res.set('Content-Length', data.length);
-      res.end(data);
-    }).catch((err) => {
-      res.status(404);
-      res.set('Content-Type', 'text/plain');
-      res.end('File not found.');
-    });
+    const contentType = mime.lookup(filename);
+    if (req.get['Range']) {
```

Comment:
Can we just check that `typeof this.adapter.getFileStream ===  'function'`

This way we'll fall through regular response for non supporting adapters and it will be forward compatible?